src/reply_maker.js

```diff
@@ -12,7 +12,7 @@ function formatReply(comment, conversions) {
   if (comment['subreddit'].match(/^totallynotrobots$/i)) {
     species = "HUMAN";
     source =  "~~SOURCE~~";
-    version = "~~" + environment['version'].toUpperCase() + "~~";
+    version = "~~" + "age"+ environment['version'].slice(1,).toUpperCase() + "~~";
```

Comment:
Hey, this looks good but the `~~` will add strikethrough markdown which is no longer needed (it used to strike through the version number), can you remove those?